xml/networking-pcipt_sriov_config.xml

```diff
@@ -115,6 +115,115 @@
    process the proper VLAN ID, which therefore has to be known in advance
   </para>
  </section>
+ <section xml:id="_pci_passthrough">
+  <title>Leveraging PCI Passthrough</title>
+  <para>
+   Two parts are necessary to leverage PCI passthrough on a &cloud; 8
+   &compnode;: preparing the &compnode;, preparing &o_comp; and &o_img;.
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     <emphasis role="bold">Preparing the &compnode;</emphasis>
+    </para>
+    <procedure>
+     <step>
+      <para>
+       There should be no kernel drivers or binaries with direct access to the
+       PCI device. If there are kernel modules, they should be blacklisted.
+      </para>
+      <para>
+       For example, it is common to have a <literal>nouveau</literal> driver
+       from when the node was installed. This driver is a graphics driver for
+       Nvidia-based GPUs. It must be blacklisted as shown in this example.
+      </para>
+<screen>&prompt.ardana;echo 'blacklist nouveau' &gt;&gt; /etc/modprobe.d/nouveau-default.conf</screen>
+      <para>
+       The file location and its contents are important; the filename is your
+       choice. Other drivers can be blacklisted in the same manner, possibly
```

Comment:
the filename is your choice? Like, what you call the file? If so, maybe rephrase to "the name of the file is your choice." Just so it's clear :)

Reply:
changed